### PR TITLE
Update for release 0.2.0

### PR DESCRIPTION
**Pull request by ciarams87**
### Proposed changes
Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue here in this description (not in the title of the PR).

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-ingress-operator/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto master
- [ ] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+### 0.2.0
```
Comment (lucacome) [inline CHANGELOG.md:3]: I think we should link to the release page, like in the other pojects

**Comment by jakuboskera** [comment]
Hi, when we can expect release 0.2.0, some ETA please? Thanks

**Comment by ciarams87** [comment]
> Hi, when we can expect release 0.2.0, some ETA please? Thanks

Hi @jakuboskera, we are working through the certification process on the RedHat side and hope to complete the release today or tomorrow!

**Review by pleshakov — APPROVED** [review]

**Review by soneillf5 — APPROVED** [review]

**Comment by jakuboskera** [comment]
> > Hi, when we can expect release 0.2.0, some ETA please? Thanks
> 
> Hi @jakuboskera, we are working through the certification process on the RedHat side and hope to complete the release today or tomorrow!

some news regarding release date? Thanks a lot

**Pull request merged**